format/index.md

```diff
@@ -300,6 +300,11 @@ pagination links, as described below.
 If a link object refers to resource objects included in the same compound
 document, it **MUST** include resource linkage to those resource objects.
 
+> Note: a resource may actually be referring to another resource returned 
+in the same compound document, but this reference may be omitted 
+from the link object of the parent resource because of sparse fieldset 
+restrictions.
+
```

Comment:
@dgeb If I'm understanding correctly, the approach you're now aiming for is the one I advocated [here](https://github.com/json-api/json-api/issues/476#issuecomment-86700323). If so, then the clause that reads "Each relationship name MUST be identical..." must also be amended, as I mentioned in that comment